lib/rules/jsx-indent.js

```diff
@@ -290,6 +297,29 @@ module.exports = {
       }
     }
 
+    /**
+     * Check indent for Literal Node or JSXText Node
+     * @param {ASTNode} node The node to check
+     * @param {Number} indent needed indent
+     */
+    function checkLiteralNodeIndent(node, indent) {
+      const value = node.value;
+      const regExp = indentType === 'space' ? /\n( *)[\t ]*\S/g : /\n(\t*)[\t ]*\S/g;
+      const nodeIndentsPerLine = Array.from(
+        matchAll(value, regExp),
+        ([, firstMatch]) => (firstMatch ? firstMatch.length : 0)
```

Comment:
oops, array destructuring doesn't work in node 4
```suggestion
        (match) => (match[1] ? match[1].length : 0)
```